src/backends/rust/parser.rs

```diff
@@ -151,7 +151,7 @@ impl<'a> FieldParser<'a> {
                         if #v != #value_type::from(#enum_id::#tag_id)  {
                             return Err(Error::InvalidFixedValue {
                                 expected: #value_type::from(#enum_id::#tag_id) as u64,
-                                actual: #v as u64,
+                                actual: 0, // TODO: #v can mutate the input and cannot be used directly.
```

Comment:
Could we fix this in a better way (without TODOs) by doing

```rust
let v = #v;
```
before the `match` and then update all occurrences of `#v` to `v` in the generated code?